bin/genome-env-next

```diff
@@ -0,0 +1,432 @@
+#!/bin/bash
+
+# if this was found via PATH as opposed to a path to executable
+THIS="$(basename "$0")"
```

Comment:
Looks like we need an `*` in there.  I expect that the `basename` version might be more obvious to non-bash authors but it seems like avoiding another process might make it worth it.